damus/Components/InvoiceView.swift

```diff
@@ -10,15 +10,28 @@ import SwiftUI
 struct InvoiceView: View {
     
     @Environment(\.colorScheme) var colorScheme
+    @Environment(\.openURL) private var openURL
     
     let invoice: Invoice
     @State var showingSelectWallet: Bool = false
     @State var inv: String = ""
+    @StateObject var user_settings = UserSettingsStore()
     
     var PayButton: some View {
         Button {
             inv = invoice.string
-            showingSelectWallet = true
+            if (user_settings.showwalletselector){
```

Comment:
nit: don't need parentheses 
`if user_settings.showwalletselector {` is sufficient